examples/logical_enclosure.rb

```diff
@@ -49,6 +49,15 @@
   action :set_script
 end
 
+# Example: Generates a support dump for the logical enclosure
+oneview_logical_enclosure 'Encl1' do
+  client my_client
+  dump_options(
+    errorCode: 'MyDump'
+  )
+  action :support_dump
```

Comment:
`support_dump` does not look like an action name. 
`create_support_dump` seems more in-line with other actions